product_configurator_mrp/tests/test_mrp.py

```diff
@@ -37,6 +39,14 @@ def setUp(self):
                 "product_qty": 1.00,
             }
         )
+        # create BOM operations line
+        self.mrpRoutingWorkcenter.create(
+            {
+                "bom_id": self.bom_id.id,
+                "name": "Operation 1",
+                "workcenter_id": self.workcenter_id.id,
+            }
+        )
```

Comment:
Kudos for touching the tests!
Now if we could only add the check that the operations were really copied, that would be the cherry on top of the cake.

Reply:
@dreispt , Yeah but testcase file is on commented.

https://github.com/OCA/product-configurator/blob/146ce493130408d5f7fa8112eaa4c65749751226/product_configurator_mrp/tests/__init__.py#L4

https://github.com/OCA/product-configurator/blob/146ce493130408d5f7fa8112eaa4c65749751226/product_configurator/tests/__init__.py#L1